moveit_core/constraint_samplers/src/constraint_sampler_tools.cpp

```diff
@@ -59,13 +61,17 @@ double constraint_samplers::countSamplesPerSecond(const ConstraintSamplerPtr& sa
 {
   if (!sampler)
   {
-    ROS_ERROR_NAMED("constraint_samplers", "No sampler specified for counting samples per second");
+    RCLCPP_ERROR(logger, "No sampler specified for counting samples per second");
     return 0.0;
   }
   robot_state::RobotState ks(reference_state);
   unsigned long int valid = 0;
   unsigned long int total = 0;
-  ros::WallTime end = ros::WallTime::now() + ros::WallDuration(1.0);
+  //TODO RCLCPP::WallTimer has not a .now() function, and there is no wallDuration (they use std::chrono::nanoseconds)
```

Comment:
@vmayoral this is something that we have to ask, they sometimes use ros::time::now() and on this case they use ros::Walltime.
ros::Walltime : _Time representation. Always wall-clock time_
In the Api of ros2, there is a [Walltimer](http://docs.ros2.org/crystal/api/rclcpp/classrclcpp_1_1WallTimer.html) but I think that it has nothing to do with ros::walltime. 

I switched it to rclcpp::Clock().now() for now, we will see if we have to change it...

Reply:
@anasarrak, can you please open an issue in ros-planning and ask for the maintainer's input?

Reply:
For tracking purposes, discussing pushed to https://github.com/ros-planning/moveit2/issues/31